src/gui/properties/trackerlistwidget.cpp

```diff
@@ -377,45 +417,31 @@ void TrackerListWidget::loadTrackers()
             oldTrackerURLs.removeOne(trackerURL);
         }
 
-        item->setText(COL_TIER, QString::number(entry.tier));
+        std::pair<int, int> peersMinMax = {-1, -1};
```

Comment:
It's better to use some particular `struct` instead of `std::pair`.

Reply:
I think it is common practice to use pair for small local pairs of data https://en.cppreference.com/w/cpp/algorithm/minmax Struct would be overkill for this purpose imho.

Reply:
Often the use of a pair does not look descriptive enough in the code. But in this case, there seems to be no problem.

Reply:
> I think it is common practice to use pair for small local pairs of data https://en.cppreference.com/w/cpp/algorithm/minmax

Returning a pair (as a special case of returning a tuple) is not so bad in modern C++, since you can always add descriptive names from the calling side:
```c++
auto [min, max] = std::minmax(//...
```

Reply:
structured bindings were added in c+17, pairs are pre c++11. It not like the usability of pairs changed, just the usage can be different. I agree accessing pair members or tuple mambers it often not readable, but in this case let's leave it.